packages/ui/react-ui/src/useWalletModal.tsx

```diff
@@ -1,11 +1,14 @@
 import { createContext, useContext } from 'react';
+import { defaultValueForModalProvider } from '@solana/wallet-adapter-react';
```

Comment:
This sets up a dependency between the `ant-design` package and the `react` package. This will be surprising to people, bloat their `node_modules` folders, and is sure to result in future pain. Let's not do this. See my thoughts on being DRY below.